apis/externalsecrets/v1alpha1/secretstore_alibaba_types.go

```diff
@@ -0,0 +1,27 @@
+package v1alpha1
```

Comment:
You need to add the copyright notice that we have on all files in the project

Reply:
(The comments on top, saying that the file is under Apache License, Version 2.0)